### PR TITLE
Add Flask server files to correct deployment location

### PR DESCRIPTION
**Pull request by krivard**
fixes the following deployment error on midas:
```
failed to deploy cmu-delphi/delphi-epidata (sgratzl/flaskx) No module named 'delphi.epidata.server.endpoints'
```

### CODE REVIEW

File: deploy.json [hunk 1]
```diff
@@ -43,14 +43,28 @@
...
+    {
+      "type": "move",
+      "src": "src/server/endpoints",
+      "dst": "[[package]]/server/endpoints/",
+      "match": "^.*\\.(py)$",
+      "add-header-comment": true
+    },
+    {
+      "type": "move",
+      "src": "src/server/endpoints/covid_utils",
```
Comment (sgratzl) [inline deploy.json:63]: isn't it called covidicast_utils? https://github.com/cmu-delphi/delphi-epidata/tree/sgratzl/flaskx/src/server/endpoints/covidcast_utils

Comment (krivard) [reply deploy.json:63]:
```suggestion
      "src": "src/server/endpoints/covidcast_utils",
```

File: deploy.json [hunk 2]
```diff
@@ -43,14 +43,28 @@
...
+      "type": "move",
+      "src": "src/server/endpoints",
+      "dst": "[[package]]/server/endpoints/",
+      "match": "^.*\\.(py)$",
+      "add-header-comment": true
+    },
+    {
+      "type": "move",
+      "src": "src/server/endpoints/covidcast_utils",
+      "dst": "[[package]]/server/covid_utils/",
```
Comment (sgratzl) [inline deploy.json:64]: same here:

```suggestion
      "dst": "[[package]]/server/endpoints/covidcast_utils/",
```

**Review by sgratzl — APPROVED** [review]
👍

**Pull request merged**